src/commands/dev/dev.js

```diff
@@ -312,6 +322,32 @@ const startPollingForAPIAuthentication = async function (options) {
   await helper(siteInfo)
 }
 
+/**
+ * @param {boolean|string} edgeInspect
+ * @param {boolean|string} edgeInspectBrk
+ * @returns {InspectSettings}
+ */
+const generateInspectSettings = (edgeInspect, edgeInspectBrk) => {
+  const enabled = Boolean(edgeInspect) || Boolean(edgeInspectBrk)
+  const pause = Boolean(edgeInspectBrk)
+  const getAddress = () => {
+    if (edgeInspect) {
+      // .slice(1) to remove the = in `=127.0.0.1:9229`
+      return typeof edgeInspect === 'string' ? edgeInspect.slice(1) : undefined
```

Comment:
I'm confused as to why you need to slice this. Commander is supposed to handle this automatically for you.

I've just pulled your branch, tried using `ntl dev --edgeInspect=127.0.0.1:1234` and `ntl dev --edgeInspect 127.0.0.1:1234` and in both cases `options.edgeInspect` has the right contents, with no need to slice.

Am I missing something?

Reply:
Aha, this was because I was (wrongfully) using the short form that [doesn't support the equal sign](https://github.com/tj/commander.js/issues/1691) ending up with `edgeInspect: '=127.0.0.1:3030'` if `ntl dev -e=127.0.0.1:1234` is used.

I've updated the help flag with more examples on how to use the short forms:
```
'netlify dev -e 127.0.0.1:9229',
'netlify dev -e127.0.0.1:9229',
'netlify dev -E 127.0.0.1:9229',
'netlify dev -E127.0.0.1:9229',
```
I don't know if this is sufficient though because the error that pops up when `-e=127.0.0.1:1234` or `-E=127.0.0.1:1234` is ran is what's below from deno. The `--help` flag suggestion will bring up the above examples though when ran with `netlify dev`.  🤔

```
error: Invalid value for '--inspect=<HOST:PORT>...': invalid IP address syntax

For more information try --help

Caused by:
    invalid IP address syntax
```